routes/index.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const router = express.Router();
+
+const User = require('../models/user');
+const constants = require('../config/constants');
+const check = require('../config/utils');
+const appName = constants.APP_NAME;
+
+let pass = {
+    title: appName,
+    header: appName,
+    welcome_message: constants.WELCOME_MESSAGE
+};
+
+/* GET home page for unregistered users */
+router.get('/', function (req, res, next) {
+    res.render('index', pass);
+});
+
+//User's sign up
+router.post('/', (req, res, next) => {
+    let username = req.body.username;
+    let password = req.body.password;
+    let email = req.body.email;
+    let phone = req.body.phone;
+
+
+    if (!username && !password && !email && !phone) {
+        pass.error_message = constants.EMPTY_PARAMS;
+        res.redirect('/')
+    }
+
+    if (!check.email(email)) {
+        pass.error_message = constants.INVALID_EMAIL;
+        res.redirect('/')
```

Comment:
missing return

Reply:
Why should there be a return if it's redirecting?